configure.ac

```diff
@@ -355,6 +355,24 @@ if test x$use_sanitizers != x; then
     ]],[[]])])
 fi
 
+dnl GCC<9.2 reports a redundant -Wsuggest-override warning on a 'final' method.
+dnl See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=78010
+AC_MSG_CHECKING(for GCC version < 9.2)
+AC_PREPROC_IFELSE([AC_LANG_SOURCE([[
+    #if !defined(__clang__) && defined(__GNUC__) && defined(__GNUC_MINOR__) && (__GNUC__ * 100 + __GNUC_MINOR__ < 902)
+    #error GCC<9.2 has the bug 78010
+    #endif
+  ]])],
+  [
+    AC_MSG_RESULT(no)
+    have_gcc_suggest_override_bug=no
+  ],
+  [
+    AC_MSG_RESULT(yes)
+    AC_MSG_WARN([GCC<9.2 reports a redundant -Wsuggest-override warning on a 'final' method, disabling -Wsuggest-override.])
```

Comment:
If we do keep this as is, let's drop this call to `AC_MSG_WARN` and the `AC_MSG_CHECKING` above. I don't think we need any extra, verbose out here. Especially since this will be the default case most people compiling going forward.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16710#issuecomment-627406029).